pom.xml

```diff
@@ -150,13 +150,18 @@
             <dependency>
                 <groupId>org.apache.tomcat.embed</groupId>
                 <artifactId>tomcat-embed-core</artifactId>
-                <version>11.0.5</version>
+                <version>11.0.6</version>
             </dependency>
             <!-- CVE in tomcat-embed-el 10.1.39 that comes with spring-boot-starter-web:jar:3.4.4 can be removed after Spring updates their dependency -->
             <dependency>
                 <groupId>org.apache.tomcat.embed</groupId>
                 <artifactId>tomcat-embed-el</artifactId>
-                <version>11.0.5</version>
+                <version>11.0.6</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.tomcat.embed</groupId>
+                <artifactId>tomcat-embed-websocket</artifactId>
+                <version>11.0.6</version>
```

Comment:
Strongly recommend using a variable in parent pom.xml to avoid version conflicts in multiple child pom.xml.